packages/google_adsense/example/integration_test/experimental_ad_unit_widget_test.dart

```diff
@@ -203,10 +225,10 @@ Future<void> pumpAdWidget(Widget adUnit, WidgetTester tester) async {
     ),
   );
 
-  // This extra pump is needed for the platform view to actually render in the DOM.
-  await tester.pump();
-  // One more for skwasm.
-  await tester.pump();
+  while (!tracker.allCalled) {
```

Comment:
If `allCalled` becomes a `Future`, this loop may be reimplemented with `Future.doWhile` or a `tracker.allCalled.then((_) { allCalled = true });` to stop the while loop.